infrastructure/src/main/java/org/corfudb/infrastructure/management/ClusterStateCollector.java

```diff
@@ -122,9 +122,12 @@ private NodeState collectLocalNodeState(long epoch, SequencerMetrics sequencerMe
         });
 
         //Build local NodeState based on pings.
-        NodeConnectivity localConnectivity = NodeConnectivity.connectivity(
-                localEndpoint, ImmutableMap.copyOf(localNodeConnections)
-        );
+        NodeConnectivity localConnectivity = NodeConnectivity.builder()
```

Comment:
good catch. 
 - Also please add a test (or an additional check) in `ClusterStateCollectorTest` which checks that the epoch is set.
 - Also would be great to have an additional check inside `NodeConnectivity` that the epoch is set. The easiest way could be to change the primitive value to a class and check that it is not null:
  ```
    @Getter
    @NonNull
    private final Long epoch;
```